docker-compose.test.yaml

```diff
@@ -7,10 +7,22 @@ services:
       target: development
     image: ffc-pay-statement-receiver-development
     container_name: ffc-pay-statement-receiver-test
+    depends_on:
+      - ffc-pay-statement-receiver-redis
     command: npm run test
     volumes:
       - ./app:/home/node/app
       - ./test:/home/node/test
       - ./test-output:/home/node/test-output
       - ./jest.config.js:/home/node/jest.config.js
       - ./package.json:/home/node/package.json
+
+  ffc-pay-statement-receiver-redis:
+    ports:
+        - "6378:6379"
```

Comment:
No need to expose ports for test container.  Will likely cause issues in Jenkins trying to bind to Jenkins localhost, particularly if port already in use.